pkg/ingester/ingester_v2.go

```diff
@@ -259,8 +271,12 @@ func (u *userTSDB) setLastUpdate(t time.Time) {
 	u.lastUpdate.Store(t.Unix())
 }
 
-// Reports tsdbIdle if TSDB can be closed, or some other tsdbIdleCheckResult.
-func (u *userTSDB) canIdleClose(idleTimeout time.Duration) (tsdbIdleCheckResult, error) {
+// Reports tsdbIdle if TSDB can be closed, or some other tsdbCloseCheckResult.
```

Comment:
> Reports tsdbIdle if TSDB can be closed

Can also be closed on `tsdbTenantMarkedForDeletion`. Maybe we should just generalise the comment and say that "Checks whether the TSDN can be closed.".